src/ChatGptNet/ChatGptClient.cs

```diff
@@ -225,4 +224,11 @@ private void UpdateHistory(Guid conversationId, IList<ChatGptMessage> messages,
 
         cache.Set(conversationId, messages, options.MessageExpiration);
     }
+
+    public async Task<IEnumerable<ChatGptMessage>> GetConversationAsync(Guid conversationId)
```

Comment:
Why don't just use something like this?

    public Task<IEnumerable<ChatGptMessage>> GetConversationAsync(Guid conversationId)
    {
        var messages = cache.Get<IEnumerable<ChatGptMessage>>(conversationId) ?? Enumerable.Empty<ChatGptMessage>();
        return Task.FromResult(messages);
    }

Reply:
I wasn't sure about the result: just tested and it works like you said